app.js

```diff
@@ -138,26 +138,36 @@ router.get('/account/:name', async ctx => {
 
 /**
  * Create a new account. Generate a throw away account id (UUID).
+ * Returns account name and public/private key.
  */
 router.post('/account', async ctx => {
     // TODO: this is using alice account to create all accounts. We may want to change that.
     const nonce = await getNonce(ctx, defaultSender);
     ctx.assert(nonce)
     const newAccountName = uuidV4();
     console.log("Creating new account " + newAccountName);
-
     // TODO: unhardcode key
+    const accountKey = hardcodedKey;
+
     const createAccountParams = {
         nonce: nonce,
         sender: await encodeAccountNameForRpc(defaultSender),
         new_account_id: await encodeAccountNameForRpc(newAccountName),
         amount: newAccountAmount,
-        public_key: hardcodedKey.public_key,
+        public_key: accountKey.public_key,
     };
 
-    const resp = await submit_transaction_rpc(client, "create_account", createAccountParams);
-    checkError(ctx, resp);
-    ctx.body = resp.result;
+    const transactionResponse = await submit_transaction_rpc(client, "create_account", createAccountParams);
+    checkError(ctx, transactionResponse);
+
+    // transactionResponse does not contain useful information, so construct a custom response
+    // TODO: is it fine to return private key here? 
```

Comment:
we don't need to return it, we need to record it as signing is handled on our side.